src/components/sideNavigation/SideNavigation.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import * as className from 'classnames';
```

Comment:
when it's a stateless component we usually write it as a pure function. your file would look like this (and it does not affect the way you use it elsewhere):

```
import * as React from 'react';
import * as className from 'classnames';
import { JSXRenderable } from '../../utils/JSXUtils'; // <-- might not be the right path

export interface ISideNavProps {
  className?: string;
  children?: JSXRenderable;
}

export const SideNavigation = (props: ISideNavProps) => 
  <nav className={className(props.className, 'navigation')}>
    <div className='navigation-menu'>
      <div className='navigation-menu-sections'>
        {props.children}
      </div>
    </div>
  </nav>
```